mentors/clojure/c.json

```
@@ -1 +1,8 @@
-[]
+[{
```

```suggestion
[
  {
```
Square brackets should appear by themselves.  Curly braces should be indented by 2 spaces.  See [README](https://github.com/exercism/website-copy/tree/master/mentors#add-your-bio)